src/Controller/UI/ReportResearchGroupDatasetStatusController.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace App\Controller\UI;
+
+use App\Form\ReportResearchGroupDatasetStatusType;
+use App\Entity\ResearchGroup;
+
+use App\Handler\EntityHandler;
+use Symfony\Component\Routing\Annotation\Route;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+/**
+ * A controller for a Report of Research Groups and related Datasets.
+ *
+ * @return Response A Symfony Response instance.
+ */
+class ReportResearchGroupDatasetStatusController extends ReportController
+{
+    // The format used to print the date and time in the report
+    const REPORTDATETIMEFORMAT = 'Y-m-d';
+
+    // The format used to put the date and time in the report file name
+    const REPORTFILENAMEDATETIMEFORMAT = 'Y-m-d';
+
+    // Limit the research group name to this to keep filename length at 100.
+    const MAXRESEARCHGROUPLENGTH = 46;
+
+    /**
+     * The default action.
+     *
+     * @param Request $request Message information for this Request.
+     * @param EntityHandler $entityHandler
+     * @param integer $researchGroupId The identifier for the Research Group subject of the report.
+     *
+     * @return Response|StreamedResponse A Symfony Response instance.
+     * @throws \Exception
```

Comment:
What throws this exception?

Reply:
I think PhpStorm added it when imported the entityHandler class, will remove it